hack/jenkins/release_build_and_upload.sh

```diff
@@ -111,5 +112,20 @@ fi
 #echo "Updating Docker images ..."
 #make push-gvisor-addon-image push-storage-provisioner-manifest
 
+echo "Generating tarball for docker images"
```

Comment:
```suggestion
echo "Generating tarballs for kicbase images"
```
nit: docker images is too broad